cloudbuild-dev.yaml

```diff
@@ -12,16 +11,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apiVersion: v1
-kind: Service
-metadata:
-  name: forseti-server-service
-  labels:
-    app: forseti-server-service
-spec:
-  ports:
-    - port: 50051
-      protocol: TCP
-  clusterIP: ${FORSETI_SERVER_IP}
-  selector:
-    app: forseti-server
\ No newline at end of file
+# This defines the build steps of Forseti docker images. The google container
```

Comment:
Can we add some comments to describe what this file specifically is for?  So that I can diferentiate the `-dev.yaml`, `-release.yaml`, and just the `cloudbuild.yaml`?